packages/create-cloudflare/e2e-tests/cli.test.ts

```diff
@@ -3,9 +3,10 @@ import { tmpdir } from "os";
 import { join } from "path";
 import { beforeEach, afterEach, describe, test, expect } from "vitest";
 import { version } from "../package.json";
+import { frameworkToTest } from "./frameworkToTest";
 import { keys, runC3 } from "./helpers";
 
-describe("E2E: Basic C3 functionality", () => {
+describe.skipIf(frameworkToTest)("E2E: Basic C3 functionality ", () => {
```

Comment:
please see my comment in [workers.test.ts](https://github.com/cloudflare/workers-sdk/pull/3836/files#diff-e79a62fdea0c7ebd9090297979b8cd9fdfe82749e6497295ee4485c2d73daf7b) which uses the same pattern.

Reply:
comment added please have a look 🙂